### ISSUE TITLE
Issue with nuxt

### ISSUE THREAD

**Issue by PiotrBaczkowski** [vue-final/vue-final-modal#9]
Hello, 
after following documentation i am getting this error:

[Vue warn]: The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside <p>, or missing <tbody>. Bailing hydration and performing full client-side render.

**Comment by PiotrBaczkowski** [comment]
however if i import like this it works 
import VueFinalModal  from 'vue-final-modal'

**Comment by hunterliu1003** [comment]
Hi @PiotrBaczkowski,

Thanks for your feedback,

I think the issue is because `vue-final-modal` using `document` for some features.

Please add `client` mode in `nuxt.config.js`:

```
plugins: [
    { src: '~/plugins/vue-stack-grid', mode: 'client' }
]
```

If it works to you, please let me know.

I will update the document soon.
In addition, I think the prop `ssr` is not needed anymore.

**Comment by PiotrBaczkowski** [comment]
hello, adding transpile fixed the issue, thanks :)

**Comment by AlbatiQue** [comment]
Hey,

I have the same issue, but the transpile solution does not seem to fix the issue in my case.

**Comment by hunterliu1003** [comment]
Hi @AlbatiQue,

Can you provide more info like below?

### Version

vue-final-modal: 
vue: 
nuxt: 

### How you use vue-final-modal

### Steps to reproduce

**Comment by AlbatiQue** [comment]
Hey @hunterliu1003 

I just did the basic setup. With the basic example for nuxt.

`<vue-final-modal
      @click-outside="clickOutside"
      @before-open="beforeOpen"
      @opened="opened"
      @before-close="beforeClose"
      @closed="closed"
    >
      ...modal content
    </vue-final-modal>

<button @click="showModal = true">
            Launch
          </button>`

"nuxt": "^2.14.0",
"vue-final-modal": "^0.19.0"

My vs file gives the following suggestion:

<img width="1028" alt="Schermafbeelding 2020-11-18 om 18 53 00" src="https://user-images.githubusercontent.com/3764232/99568171-638baf80-29cf-11eb-92ab-3de5433dd6d3.png">

**Comment by Mini-ghost** [comment]
@AlbatiQue 
I have a working solution, that is working in my project very well.

Create an `index.d.ts` file in root, and add this code block below:

```d.ts
// index.d.ts
declare module 'vue-final-modal/lib' {
  import VueFinalModal from 'vue-final-modal'
  export default VueFinalModal
}
```

Hope it useful 😉

**Comment by hunterliu1003** [comment]
Bugfix was released in [v0.21.2](https://www.npmjs.com/package/vue-final-modal/v/0.21.2).
